### ISSUE TITLE
Support it.only or iit

### ISSUE THREAD

**Issue by juliemr** [juliemr/minijasminenode#1]
These features will be available in Jasmine 2.0, but it would be nice to have them in the meantime.

**Comment by searls** [comment]
Hey @juliemr. It might interest you to know that I just pushed a working [jasmine-only](https://github.com/davemo/jasmine-only) to npm and it works in conjunction with minijasminenode just fine (one just needs to do `require('jasmine-only') from a spec helper so that it can monkey-patch the global jasmine goods).

The output is a little silly, though. Here's a suite with 4 specs but only the third one is actually being run with `it.only` (see "4 tests, 1 assertion")?

```
....

Finished in 0.004 seconds
4 tests, 1 assertion, 0 failures
```

So that is to say it "works" now, but it's not abundantly obvious to the user that it's working. I suspect your custom reporter would just need to be tweaked to exclude non-asserting tests.

If you need an example feel free to ask

**Comment by juliemr** [comment]
Awesome, thanks for this workaround!

**Comment by juliemr** [comment]
Closed in 0.2.6
